README.md

```diff
@@ -1 +1,34 @@
 # trails-review
+
+prepare portfolio website
+# Additional description about the project and its features.
+
+This project Is for setting header and headline section to portfolio website using semantic HTML tags and clean style file
```

Comment:
add more descriptive lines to your project

Reply:
you need to add descriptive lines so you can make it more clear for anybody who is gonna see it